verification/curator-service/api/src/controllers/sources.ts

```diff
@@ -26,9 +26,14 @@ export default class SourcesController {
             res.status(422).json('limit must be > 0');
             return;
         }
+        const filter = req.query.url
```

Comment:
would be nice to augment the open api spec with this new query param (fine in another PR)

Reply:
Good call, done.